assets/css/global.less

```diff
@@ -1401,6 +1401,9 @@ body .guides .guides-section-white {
 
 .post-bg{
   background-color: #f6f8fa; 
+  .container-fluid {
+    max-width: 1200px;
```

Comment:
What's the intention of this line? It looks like this is causing an issue with the post width (it shrunk considerably on my screen)

Reply:
Before this commit:
<img width="1659" alt="Screen Shot 2019-09-09 at 18 16 05" src="https://user-images.githubusercontent.com/1516418/64552582-6f8dd300-d32f-11e9-8be7-fb9def796621.png">

After this commit:
<img width="1657" alt="Screen Shot 2019-09-09 at 18 22 04" src="https://user-images.githubusercontent.com/1516418/64552586-71579680-d32f-11e9-8115-030221f67208.png">

Reply:
I wanted to maintain a max-width for both detail and listings page so it doesn't stretch so much on larger devices. The listings page also has a max-width of 1200px.

Reply:
My device is not large though, I'm on a 13' macbook, before my screen was matching the mock and now it is not anymore. Since this is causing some screens to be buggy and this is not related to "table styling" anyway, I suggest you to remove this from this PR, and we discuss this separately. I also think we should get a browserstack subscription and see how it works on our screens!

Reply:
We do need a max width, as on my 1920x1080 screen, it ends up super wide, but +1 on doing that in a separate PR.

Reply:
Task created here to fix this https://gruntwork.atlassian.net/browse/HSTN-317

Reply:
Awesome! Once this is removed from this PR then I think this PR is good to go then. Or is there anything else pending?